view/frontend/web/template/payment/offsite-alipay-form.html

```diff
@@ -8,9 +8,13 @@
                value: getCode(),
                checked: isChecked,
                click: selectPaymentMethod,
-               visible: isRadioButtonVisible()" />
+               visible: isRadioButtonVisible(),
+               enable: isActive()" />
         <label data-bind="attr: {'for': getCode()}" class="label">
             <span data-bind="text: getTitle()"></span>
+            <span 
+                data-bind="i18n: '(option available only for orders in Thai baht)',
```

Comment:
@jacekstan Uhmm I'm still 50/50 on this.
Because in fact, it doesn't matter what your order's currency is. The store's currency has to be the same.

i.e. if store currency is set to `usd` and buyer choose to order in `thb`, it still won't work with this condition.

Maybe we just really need to hide the payment method if store currency isn't THB?

Reply:
I agree but I also think that would be good question to PM ?

Reply:
@guzzilar as agreed with Big, we are keeping option, but disabled.